docs/building-a-form/creating-a-form-config-file.md

```diff
@@ -1,286 +1,173 @@
 # Creating a form config file
 
-Your form is generated from a JSON Schema configuration file called `form.js`, along with a few other key configuration files. For information about the basic files you need to configure a form, see "Create required files" in the *US Forms System Getting Started Guide*.
+Your form is generated from a JSON Schema configuration file called `form.js`, along with a few other key configuration files.
 
-### In this guide
-
-- [Example `form.js` file](#example-form.js-file)
-- [About the `schema` and `uiSchema` objects](#about-the-schema-and-uischema-objects)
-- [Configuring `uiSchema` using rjsf options](#configuring-uischema-using-rjsf-options)
-- [Configuring `uiSchema` using US Form System options](#configuring-uischema-using-us-form-system-options)
-
-### Example `form.js` file
-
-This example `form.js` file will get you started with building your form.
-
-```js
-{
-  // Prefix string to add to the path for each page.
-  urlPrefix: '',
-
-  // The introduction page component. To exclude an introduction page, remove this component.
-  introduction: IntroductionComponent,
-
-  // The confirmation page component that will render after the user successfully submits the form.
-  confirmation: ConfirmationComponent,
+- For information about the basic files you need to configure a form, see "[Create required files](../getting-started/installing-the-us-forms-system-in-an-existing-application.md#create-required-files)."
+- For an example `form.js` file, see "[Quick Start: Example `form.js` file](quick-start-example-formjs-file.md)"
 
-  // The prefix for Google Analytics events that are sent for different form actions.
-  trackingPrefix: '',
-
-  // The title of the form, rendered on all pages.
-  title: '',
-
-  // The subtitle of the form, usually the form number. The subtitle is rendered on all pages when there's also a title.
-  subTitle: '',
+### In this guide
 
-  // Schema definitions that can be referenced on any page. These are added to each page's schema in the reducer code, so that you don't have to put all of the common fields in the definitions property in each page schema. For more information on definitions, see schema.definitions below.
-  defaultDefinitions: {},
+- [About the `Form` component, fields, and widgets](#about-the-form-component-fields-and-widgets)
```

Comment:
At this point, these are the only sections in "Creating a form config file". Is this a reasonable grouping of information? What should the title be? Should the widgets topic ("[Adding widgets and definitions](https://github.com/usds/us-forms-system/blob/schema-uischema/docs/building-a-form/adding-widgets-and-definitions.md#adding-widgets-and-definitions)") move into this topic?

Reply:
I think I would keep a separate file on widgets, because this one is already quite long. But I might also separate out the "Adding widgets and definitions" file into 2, one for widgets and one for definitions.

Reply:
> I might also separate out the "Adding widgets and definitions" file into 2, one for widgets and one for definitions.

Excellent, that seems ideal. 👍